scripts/gen_eval_files.lua

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env -S nvim -l
+
```

Comment:
No: e0d97d264f83d45472e6cdd1051893db1083eefb

Reply:
The build scripts will still use the built binary. This is just so I can develop the script without invoking the cmake jet engine.

Reply:
Eh, fair, it should still work.

Reply:
I can confirm because `make doc` still works.